docs/development/classlib/classlib-manual.rst

```diff
@@ -0,0 +1,72 @@
+.NET Core Class Library Manual
+==============================
+
+.NET Core is a modern implementation of .NET. It includes many familiar .NET APIs, which are supported on Linux, OS X, Windows and other operating systems. It is both an implementation but also a standard API definition that can (and is) supported by other versions of .NET. 
+
+.NET Core can be reasonably thought of as the next version of Portable Class Libraries (PCL). This means that code compiled for .NET Core can run on both .NET Core and other .NET versions that support the .NET Core API, such as the .NET Framework. For library developers, it can be a compelling API to target for supporting multiple platforms. 
+
+This document will describe how to create .NET Core class libraries, both from the commandline and with Visual Studio. It will also explain how to create NuGet packages that support one or more platforms.
+
+Targeting .NET Platforms
+========================
+
+Library developers often target one of more .NET platforms. The following table lists the set of platforms that you can target with .NET Core and/or PCL. There is an assumed column missing, which is explicitly targeting a platform. That choice is always available, and is the only choice in the case of the .NET Framework 3.5, as listed below.
+
++----------------------+------+-----+
+| Platform             | Core | PCL |
++======================+======+=====+
+| .NET Core            | x    | x   |
++----------------------+------+-----+
+| DNX Core 50          | x    | x   |
++----------------------+------+-----+
+| UWP 10               | x    | x   |
++----------------------+------+-----+
+| .NET Framework 4.5+  | x    | x   |
++----------------------+------+-----+
+| WP 8.1               | x    | x   |
++----------------------+------+-----+
+| WPS 8.x              | x    | x   |
++----------------------+------+-----+
+| Xamarin.*            |      | x   |
++----------------------+------+-----+
+| Mono                 |      | x   |
++----------------------+------+-----+
```

Comment:
If by .NET Core class library you mean building a library by hand that just references packages, it can also target Xamarin and Mono as both support contracts for which we have packages.